donate/core/models.py

```diff
@@ -241,17 +241,32 @@ class CampaignPage(DonationPage):
         models.PROTECT,
         related_name='+',
     )
+
     lead_text = models.CharField(max_length=800)
+    donate_form_header = models.CharField(max_length=50, blank=True, default="")
+    intro_header = models.CharField(max_length=200, blank=True, default="Donate now")
     intro = RichTextField()
 
     content_panels = Page.content_panels + [
+        FieldPanel('lead_text'),
         FieldPanel('project'),
         ImageChooserPanel('hero_image'),
-        FieldPanel('lead_text'),
+        FieldPanel('donate_form_header'),
+        FieldPanel('intro_header'),
         FieldPanel('intro'),
         InlinePanel('donation_amounts', label='Donation amount overrides'),
     ]
 
+    cta_first = models.BooleanField(
```

Comment:
From mockup it looks like we want to always keep CTA on the right on desktop regardless. If I'm not misunderstanding it then let's update the help text so it provides more clarity to users.

Reply:
Yep that's correct, I forgot to change the text here after the `order` adjustments